src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.scss

```diff
@@ -1,3 +1,26 @@
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+@import '~bootstrap/scss/grid';
```

Comment:
Please don't import bootstrap in any component scss, as this greatly increases the size of the frontend build.
Recently we merged a PR (https://github.com/ceph/ceph/pull/34283) that adds a new file where all the scss that requires boostrap should be used:
src/styles/bootstrap-extends.scss
Please move just the styles that need bootstrap to this file.

```suggestion
```

Reply:
> Please don't import bootstrap in any component scss, as this greatly increases the size of the frontend build.
> Recently we merged a PR (#34283) that adds a new file where all the scss that requires boostrap should be used:
> src/styles/bootstrap-extends.scss
> Please move just the styles that need bootstrap to this file.

@tspmelo  Please check.